app/deepseg.cc

```diff
@@ -95,6 +96,8 @@ int main(int argc, char* argv[]) try {
 			flipHorizontal = !flipHorizontal;
 		} else if (args[arg] == "-V") {
 			flipVertical = !flipVertical;
+		} else if (args[arg] == "-M") {
+			multipass = !multipass;
```

Comment:
This toggles the multipass flag if `-M` is used multiple times.. intended or confusing?

Reply:
That's the same as with the other existing flags …